tensorboard/plugins/audio/summary_test.py

```diff
@@ -250,6 +250,39 @@ def test_default_step(self):
             # Reset to default state for other tests.
             tf2.summary.experimental.set_step(None)
 
+    def test_lengths(self):
+        data = np.array([[[1.0], [0.0]], [[1.0], [-1.0]]]).astype(np.float32)
+        lengths = np.array([1, 2]).astype(np.int32)
+        event = self.audio_event("a", data, 44100, step=333, lengths=lengths)
```

Comment:
Maybe fine to just use `pb = self.audio(...)` here? We don’t really care
about hte step or wall time (the tests that use `audio_event` are
specifically testing that behavior), and this makes it slightly easier
to pluck out the tensor proto.

Reply:
Done.